kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -132,16 +175,33 @@
 
   /**
    * Create a {@link Dataset} for the given dataset or view URI.
+   * {@code create} returns an empty dataset. You can use 
+   * {@link DatasetWriter} to populate your dataset.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * URIs must begin with {@code dataset:} or {@code view:}. The remainder of
+   * the URI is implementation specific, depending on the dataset scheme.
```

Comment:
If the URI is a view URI, this method creates the underlying dataset and returns a view of it.